package.json

```diff
@@ -161,7 +162,7 @@
     "rimraf": "^3.0.2",
     "ts-jest": "^25.4.0",
     "ts-node": "^8.8.2",
-    "typescript": "^3.9.0",
+    "typescript": "^4.6.2",
```

Comment:
Is TypeScript needed at the root level? There are TS files in the `build` directory but I don't know if they're in use.

Reply:
Noted above

Reply:
There are TS files that are used by Beachball in `./build/releasing` folder as well.